commit 90a2e69e3d4ef8e13f0b6b2d44818299a7f354cd
Author: Tim Foster <tim.foster@joyent.com>
Date:   2019-05-15T11:33:07+01:00 (5 months ago)
    
    TOOLS-2230 sdc-amon build fails due to missing zutil configure

diff --git a/Makefile b/Makefile
index 985b52a..5255d77 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@
 #
 
 #
-# Copyright (c) 2019, Joyent, Inc.
+# Copyright 2019 Joyent, Inc.
 #
 
 #
@@ -85,7 +85,7 @@ endif
 # an agent installation or uninstallation, so we provide a magic environment
 # varible to disable them here.
 #
-NPM_ENV = SDC_AGENT_SKIP_LIFECYCLE=yes MAKE=$(MAKE)
+NPM_ENV = npm_config_cache=$(TOP)/$(BUILD)/.npm SDC_AGENT_SKIP_LIFECYCLE=yes MAKE=$(MAKE)
 
 NODE_DEV := ./node_modules/.bin/node-dev
 TAP := ./node_modules/.bin/tap
